x-pack/plugins/siem/public/alerts/containers/detection_engine/alerts/translations.ts

```diff
@@ -6,29 +6,29 @@
 
 import { i18n } from '@kbn/i18n';
 
-export const SIGNAL_FETCH_FAILURE = i18n.translate(
-  'xpack.siem.containers.detectionEngine.signals.errorFetchingSignalsDescription',
+export const ALERT_FETCH_FAILURE = i18n.translate(
+  'xpack.siem.containers.detectionEngine.alerts.errorFetchingAlertsDescription',
   {
-    defaultMessage: 'Failed to query signals',
+    defaultMessage: 'Failed to query alerts',
   }
 );
 
 export const PRIVILEGE_FETCH_FAILURE = i18n.translate(
-  'xpack.siem.containers.detectionEngine.signals.errorFetchingSignalsDescription',
+  'xpack.siem.containers.detectionEngine.alerts.errorFetchingAlertsDescription',
   {
-    defaultMessage: 'Failed to query signals',
+    defaultMessage: 'Failed to query alerts',
   }
 );
 
 export const SIGNAL_GET_NAME_FAILURE = i18n.translate(
-  'xpack.siem.containers.detectionEngine.signals.errorGetSignalDescription',
+  'xpack.siem.containers.detectionEngine.alerts.errorGetAlertDescription',
   {
     defaultMessage: 'Failed to get signal index name',
```

Comment:
Should this (and the one below it) be changed? I saw your note about not touching `signalsIndex` but I see other similar strings being changed so 🤷

Reply:
Yeah it was tough to draw the line with some of these, but I left this and the one below as is since it was referencing the signals index and we're not quite sure what that's going to be named yet.